examples/browser-webrtc/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+authors = ["Doug Anderson <douganderson444@peerpiper.io>"]
+edition = "2021"
+license = "MIT"
+name = "browser-webrtc-example"
+publish = false
+rust-version = { workspace = true }
+version = "0.1.0"
```

Comment:
Getting the following warning: 

```
[INFO]: Optional fields missing from Cargo.toml: 'description', 'repository'. These are not necessary, but recommended
```

Not critical but worth addressing to reduce noise?

Reply:
Added!